cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -184,6 +196,10 @@ data:
             proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
         }
 
+        location /pkg.service.allocation.v1/ {
```

Comment:
same question whether all services will share 'grpcserver'

Reply:
I think all services should share grpc server, so we likely want to make the endpoint `/` if we can? @avrodrigues5 can we just make it:
```
location / {
...
}
```

?